rust/src/lz4_binding.rs

```diff
@@ -0,0 +1,46 @@
+extern crate lz4;
```

Comment:
Personally, I am not a big fan of the `lz4_binding` name. We now have a stack of `lz4-sys`, `lz4` and `lz4_binding`.

If we could define a `wkwrap::lz4` submodule, I'd prefer that. But, obviously, that's not a deal-breaker.